src/platform-includes/getting-started-verify/apple.mdx

```diff
@@ -20,3 +20,5 @@ if (error) {
     [SentrySDK captureError:error];
 }
 ```
+
+To verify crashes, ensure you run your application without a debugger attached. Otherwise, the SDK will not capture the crash.
```

Comment:
```suggestion
<Note>

To verify crashes, ensure you run your application without a debugger attached. Otherwise, the SDK will not capture the crash.

</Note>
```
wanna put that as a note perhaps?